components/form/form.css

```diff
@@ -0,0 +1,49 @@
+.main__form {
```

Comment:
I don't understand how this element relates to the form regarding BEM, could you please explain?

Reply:
You're right. It has no relation =) I changed it to `.main-form`.